helpful.py

```diff
@@ -67,7 +67,30 @@ class Helpful:
 
     def __init__(self, bot):
         self.bot = bot
-
+        self.port = bot.listen('on_message')(self.port)
+
+    @bot.command(pass_context = True)
+    async def mod(self, ctx, rule_num, *, link):
+        """!mod <rule number> <link to objectionable message>"""
+        FEU_id = "144670830150811649"
+        if ctx.message.server is None or ctx.message.server.id == FEU_id:
+            await self.bot.send_message(ctx.message.author, "Your request for moderation was successful.")
+            if ctx.message.server is not None:
+                await self.bot.delete_message(ctx.message)
+            mod_channel = self.bot.get_channel("650911156277477377")
+            paladins = discord.utils.get(ctx.message.server.roles, id="145992793796378624").mention
+            await self.bot.send_message(mod_channel, "%s, moderation request received by user %s: Rule %s, at <%s>." % (paladins, ctx.message.author.name, rule_num, link))
+        else:
+            await self.bot.say("Moderation features are for FEU only.")
+            
+    @bot.command()
+    async def howtomod(self):
```

Comment:
why is this present in both `helpful.py` and also `feubot.py`?

Reply:
I forgot to erase those lines, I'll do that soon. For the split commands into other .py files, the cogs that would reference and allow for the commands to be run wouldn't work so I put them all into the main file, that is why there are duplicates for the others like helpful.py. I haven't had the chance to look into cogs or splitting, so I just left them as is.